### PR TITLE
TBSTFNP-21: Document minimum TFS agent software

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

File: doc/analyze-from-tfs.md [hunk 1]
```diff
@@ -140,6 +140,9 @@ Try modifying the build definition to remove the SonarQube.MSBuild.Runner.exe en
 
 Most analysis-related configuration or execution errors will cause the build to fail and will be appear on the Build Summary. Additional information can be found by viewing the logs or diagnostic information (i.e. by clicking on **View Log**, or **Diagnostics** at the top of the Build Summary page).
 
+#### During a XAML build, the analysis fails with a "System.ArgumentNullException ... Parameter name: tfsUri"
+
+Make sure you are using at least TFS 2013 Update 2 agent software.
```
Comment (duncanp-sonar) [inline doc/analyze-from-tfs.md:145]: There as section above on "Installing the TFS 2013 Object Model on a TFS 2015 Build Agent". Does it matter which version of the OM is installed? (possibly not if the issue is with the agent).

**Comment by duncanp-sonar** [comment]
LGTM (although TBSTFNP-21 is the transparent icons in the build task bug)

File: doc/analyze-from-tfs.md (hunk 1 above)
Comment (bgavrilMS) [inline doc/analyze-from-tfs.md:145]: As far as I can tell, there is only one version of the TFS 2013 OM (https://visualstudiogallery.msdn.microsoft.com/3278bfa7-64a7-4a75-b0da-ec4ccb8d21b6). I've tested on TFS 2015 Agents + this OM in the past.

**Comment by bgavrilMS** [comment]
Good point, I'll change the commit message.

File: doc/analyze-from-tfs.md (hunk 1 above)
Comment (duncanp-sonar) [inline doc/analyze-from-tfs.md:145]: Following the instructions in the doc (i.e. "Search for Team Foundation Server Object Model") finds three versions for 2013: the one you listed and ones for updates 4 and 5. However, if it works with the lowest update version then it should be fine.

**Pull request closed without merging**